### PR TITLE
Updated MDK tutorials

### PR DESCRIPTION
**Pull request by jitendrakansal**
Updated MDK tutorials

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md [hunk 1]
```diff
@@ -483,48 +483,58 @@ Since you will display the customer's address in Google Maps on Android device,
...
 
-[OPTION BEGIN [Android]]
 
-In this step, you will run the app on an android device.
-
-1. Navigate to `/DemoSampleApp.mdkproject/App_Resources/Android/app.gradle`.  
+      !![MDK](img-7.1.png)
 
-    !![MDK](img_8.1.png)
+    >Files specifying in the `.mdkproject/App_Resources_Merge` folder overrides a part of the resource files in `<generated-project>/app/App_Resources`. You can find more details about it in [help documentation](https://help.sap.com/doc/f53c64b93e5140918d676b927a3cd65b/Cloud/en-US/docs-en/guides/getting-started/mdk/custom-client/app-resources-merge.html).
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md:502]: Please check the first sentence. It is looking weird. May be a grammar issue.

File: tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md [hunk 2]
```diff
@@ -483,48 +483,58 @@ Since you will display the customer's address in Google Maps on Android device,
...
-    !![MDK](img_8.3.png)
 
-   4. Provide below information before `application` closing tag.
+[ACCORDION-BEGIN [Step 8: ](Run the MDK Client)]
+
+>Make sure you are choosing the right device platform tab above.
+
+[OPTION BEGIN [Android]]
+
+In this step, you will run the app on an android device.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md:522]: Android

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MichaelCzcz** [comment]
Did make any changes to line 502? The first sentence ("Files specifying in the...") still look weird. Did you want to say "specified"?

**Comment by jitendrakansal** [comment]
@MichaelCzcz updated if say so. Please check

**Pull request merged**